pkg/kubelet/cm/cpumanager/policy_static.go

```diff
@@ -29,8 +29,12 @@ import (
 	"k8s.io/kubernetes/pkg/kubelet/cm/topologymanager/bitmask"
 )
 
-// PolicyStatic is the name of the static policy
-const PolicyStatic policyName = "static"
+const (
+	// PolicyStatic is the name of the static policy
+	PolicyStatic policyName = "static"
+	// SMTAwarePolicyOption is the name of the CPU Manager policy option
+	SMTAwarePolicyOption string = "smtaware"
```

Comment:
I probably should have brought this up earlier, but I'm not sold on the name `smtaware` for this option.

The static policy is already technically `smtaware`.  In fact, this option actually leverages the fact that it is already `smtaware` to ensure that it allocates all hyper-threads from a single-cpu before considering hypethreads from other cpus.

This option simply add a restriction on top of this to only admit pods that request cores in multiples of the number of hyperthreads per cpu (thus forcing the existing policy to allocate whole-cpus).

I don't have a good suggestion for a different name off the top of my head, but I think we want something more descriptive than just `smtaware`.

Reply:
Yes, this is totally true and a valid point. `smtaware` is the internal name we picked up initially when we started researching and of course it stuck (naming is hard).
I think we can just replace `smtaware` with something more accurate like `smtforce` or `smtalign` or `smtforcealign`.
Thinking about it, we will need to check if the user-facing value shoud be `camelCase` (`SMTForceAlign`) or not.